docs/book/src/topics/external-cloud-provider.md

```diff
@@ -1,6 +1,6 @@
 # External Cloud Provider
 
-To deploy a cluster using [external cloud provider](https://github.com/kubernetes/cloud-provider-openstack), create a cluster configuration with the [external cloud provider template](../templates/cluster-template-external-cloud-provider.yaml).
+To deploy a cluster using [external cloud provider](https://github.com/kubernetes/cloud-provider-openstack), create a cluster configuration with the [external cloud provider template](../../../../templates/cluster-template-external-cloud-provider.yaml).
```

Comment:
Tbh, I don't think that this is the correct solution, as the book will/can be build.

```suggestion
To deploy a cluster using [external cloud provider](https://github.com/kubernetes/cloud-provider-openstack), create a cluster configuration with the [external cloud provider template](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/templates/cluster-template-external-cloud-provider.yaml).
```

Maybe this would be better?

Reply:
WDYT @jichenjc @hidekazuna

Reply:
Same for e.g., [docs/book/src/clusteropenstack/configuration.md](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/book/src/clusteropenstack/configuration.md)

Reply:
@tobiasgiese Thanks. I actually build book and confirmed original solution does not work.

Reply:
thanks~ , I didn't notice this as we used to have such link before , will update :)

Reply:
But I don‘t think that my suggestion is correct either. Do we publish the book somewhere?

Reply:
so far ,not yet, but we do have a plan though not execute it ...